pom.xml

```diff
@@ -235,6 +235,7 @@
                      <exclude>parquet/org/**</exclude>
                      <exclude>parquet/column/**</exclude>
                      <exclude>parquet/hadoop/ParquetInputSplit</exclude>
+                     <exclude>parquet/thrift/struct/**</exclude>
```

Comment:
I don't need this exclusion anymore, I kept the deprecated constructor, which is never used internally but if someone uses it externally it'll still work.